src/test/integrasjonstester/kotlin/no/nav/familie/ba/sak/kjerne/beregning/domene/TilkjentYtelseRepositoryTest.kt

```diff
@@ -69,4 +75,48 @@ class TilkjentYtelseRepositoryTest(
             assertThat(harFagsakTattIBrukNyKlassekodeForUtvidetBarnetrygd).isFalse()
         }
     }
+
+    @Nested
+    inner class FindByFagsak {
+        @Test
+        fun `skal returnere alle tilkjente ytelser tilknyttet fagsak`() {
+            // Arrange
+            val aktør = aktørIdRepository.save(randomAktør())
+            val fagsak = fagsakRepository.save(lagFagsak(aktør = aktør))
+            val behandling = behandlingRepository.save(lagBehandling(fagsak = fagsak))
+            val migreringsdatoPluss1Mnd = LocalDate.of(2025, 1, 1)
+
+            tilkjentYtelseRepository.save(
+                lagTilkjentYtelse(
+                    behandling = behandling,
+                    utbetalingsoppdrag = objectMapper.writeValueAsString(lagUtbetalingsoppdrag(listOf(lagUtbetalingsperiode(behandlingId = behandling.id, periodeId = 1, forrigePeriodeId = null, ytelseTypeBa = YtelsetypeBA.ORDINÆR_BARNETRYGD, opphør = Opphør(migreringsdatoPluss1Mnd))))),
+                ),
+            )
+
+            // Act
+            val tilkjenteYtelser =
+                tilkjentYtelseRepository.findByFagsak(
+                    fagsakId = fagsak.id,
+                )
+
+            // Assert
+            assertThat(tilkjenteYtelser).hasSize(1)
+        }
+
+        @Test
+        fun `skal returnere false dersom det ikke er sendt opphør fra migreringsdato pluss 1 mnd tidligere`() {
+            // Arrange
+            val aktør = aktørIdRepository.save(randomAktør())
+            val fagsak = fagsakRepository.save(lagFagsak(aktør = aktør))
+
+            // Act
+            val tilkjenteYtelser =
+                tilkjentYtelseRepository.findByFagsak(
+                    fagsakId = fagsak.id,
+                )
+
+            // Assert
+            assertThat(tilkjenteYtelser).hasSize(0)
```

Comment:
Småplukk, men kan vurdere 

```suggestion
            assertThat(tilkjenteYtelser).isEmpty()
```